modules/analysis/templates/AOMDailyUsers.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+URL_PATTERN="https://addons.mozilla.org/en-US/firefox/addon/adblock-plus/statistics/usage-day-%s-%s.csv"
+NOW=$(date +"%Y%m%d")
+LOG_PATH=<%= @log_dir %>
+LOG_FILE="$LOG_PATH/AOMDailyUsers.log"
+
+if [ -f $LOG_FILE ]
```

Comment:
Please handle just one date (or a range) based on options (e.g. via `getopt`) - so one can define which time-frame to fetch. Without any arguments or options, the script should return the current values..